### PR TITLE
LPS-153451 portal-search-rest-test: Add Backend Test Coverage for suggestions REST

### PR DESCRIPTION
**Pull request by gustavolimav**
https://liferay.atlassian.net/browse/LPS-162242

### CODE REVIEW

File: modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java [hunk 1]
```diff
@@ -6,14 +6,123 @@
...
 /**
  * @author Petteri Karttunen
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class SuggestionResourceTest extends BaseSuggestionResourceTestCase {
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
```
Comment (peerkar) [inline modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java:35]: The testGroup is available from the base class so no need to create it here. The base class also cleans it up.

File: modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java [hunk 2]
```diff
@@ -6,14 +6,123 @@
...
+	@Test
+	public void testPostSuggestionsPage() throws Exception {
+		String displayGroupName = "Suggestions";
+
+		String title = "Document Title";
+
+		_addJournalArticle(title);
+
+		SuggestionsContributorConfiguration[]
+			suggestionsContributorConfigurations = {
```
Comment (peerkar) [inline modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java:51]: Inline to line 58

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:search

File: modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java [hunk 3]
```diff
@@ -81,7 +67,10 @@ public void testPostSuggestionsPage() throws Exception {
 	}
 
 	private JournalArticle _addJournalArticle(String title) throws Exception {
-		return JournalTestUtil.addArticle(_group.getGroupId(), title, "");
+		return JournalTestUtil.addArticle(
+			GroupTestUtil.addGroup(
```
[the remark this answers is not in the document]
Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java:71]: forgot to change it :p

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:forward

**Pull request closed without merging**